config/config.go

```diff
@@ -116,6 +116,7 @@ var (
 	DefaultConsulSDConfig = ConsulSDConfig{
 		TagSeparator: ",",
 		Scheme:       "http",
+		WatchTimeout: 30 * time.Second,
```

Comment:
This is what the setting was before (for the actual implementation, not the tests). So I thought to not change it.